System/Posix/Files.hsc

```diff
@@ -104,6 +104,13 @@ import System.Posix.Internals
 
 import Data.Time.Clock.POSIX (POSIXTime)
 
+-- throwErrnoTwoPathsIfMinus1_
+--
+-- | For operations that require two paths (e.g., renaming a file)
+throwErrnoTwoPathsIfMinus1_ :: (Eq a, Num a) => String -> FilePath -> FilePath -> IO a -> IO ()
+throwErrnoTwoPathsIfMinus1_ loc path1 path2 =
+  throwErrnoIfMinus1_ (loc <> " '" <> path1 <> "' to '" <> path2 <> "'")
```

Comment:
I should probably have mentioned that `(<>)` is not in `Prelude` prior to GHC 8.4, but we support 8.2, so you'll need:
```haskell
#if !MIN_VERSION_base(4,11,0)
import Data.Semigroup ( (<>) )
#endif

Reply:
Yeah, I just ran into that :) I will push a new update in a few ...